dns_test.go

```diff
@@ -476,7 +477,7 @@ func TestGetDNSRecord(t *testing.T) {
 			"auto_added": true,
 			"source":     "primary",
 		},
-		Comment: "This is a comment",
+		Comment: &comment,
```

Comment:
we can use [`StringPtr`](https://github.com/cloudflare/cloudflare-go/blob/master/convert_types.go#L571) to simplify this here